plugins/plugin-sass/plugin.js

```diff
@@ -5,6 +5,7 @@ const npmRunPath = require('npm-run-path');
 
 const IMPORT_REGEX = /\@(use|import|forward)\s*['"](.*?)['"]/g;
 const PARTIAL_REGEX = /([\/\\])_(.+)(?![\/\\])/;
+const NODE_MODULES_PATH = path.join(__dirname, '..', '..'); // 2 dirs up from node_modules/@snowpack/plugin-sass/. This is a guess, but it’s a last-resort guess, so it’s OK if it’s wrong.
```

Comment:
this makes me nervous... can we use `find-up` instead?

Reply:
I was a bit worried that if we used `find-up`, it may start resolving things in a different `node_modules` folder than where this plugin is installed. The reasoning was that wherever this specific project’s `@snowpack/plugin-sass` is installed, that’s the only `node_modules` folder we should be trying to resolve from. This should work for single projects or workspaces since `node_modules` would be shared.

Is there a scenario where a project pulls from multiple `node_modules` locations? Or where this one would be isolated from the rest? I just didn’t want to add a dependency to this package without solving for something, or having this start to look in parent directories it shouldn‘t be.

Reply:
Yarn 2 jumps to mind. Also workspaces where things can be oddly linked.

Maybe instead, use `find-up` from the `config.root`? That way, you're finding up in the user's project. So if for some reason nesting was an issue, you'd be sure to get the exact version that the user had installed themselves.

Reply:
WFM! Thanks for clarifying. Yeah that should be a bit more reliable.

Reply:
And yes, you‘re right aliasing will be weird. But that‘s something people will just have to deal with since it’s something they set up themselves. Also, there is still the `loadPaths` option in this plugin that will always happen before the node_modules search so any weird alias setup is still supported, even with this PR.

Reply:
Added, and tested it locally. Still seems to be resolving the same, at least in a “typical“ setup